<launch>
    <arg name="car"/>
    <group if="$(eval car == 1)">
        <include file="$(find dbw_polaris_can)/launch/dbw.launch" />
        <include file="$(find velodyne_pointcloud)/launch/VLP16_points.launch">
            <arg name="laserscan_ring" value="5"/>
        </include>
        <include file="$(find piksi_multi_rtk_ros)/launch/piksi_multi_rover.launch">
            <arg name="interface" value="tcp" />
            <arg name="tcp_addr" value="192.168.0.222" />
            <arg name="load_enu_origin_from_file"   value="true" />
            <arg name="enu_origin_file"             value="$(find ltu_actor_vehicle_gem)/enu_origin.yaml"/>
        </include>
    </group>
    <group if="$(eval car == 2)">
        <include file="$(find dbw_polaris_can)/launch/dbw.launch" />
        <include file="$(find velodyne_pointcloud)/launch/VLP16_points.launch">
            <arg name="device_ip" default="192.168.60.50" />
            <arg name="laserscan_ring" value="12"/>
        </include>
        <include file="$(find piksi_multi_rtk_ros)/launch/piksi_multi_rover.launch">
            <arg name="interface" value="tcp" />
            <arg name="tcp_addr" value="192.168.60.30" />
            <arg name="load_enu_origin_from_file" value="true" /> 
            <arg name="enu_origin_file" value="$(find actor2_support)/config/piksi/enu_origin.yaml"/>
        </include>
    </group>

    <node name="waypoint" pkg="ltu_actor_route_waypoint" type="waypoint">
        <param name="speed" value="2.5"/>
        <param name="gps_fix" value="/piksi/navsatfix_best_fix"/>
        <param name="gps_vel_ned" value="/piksi/vel_ned"/>
    </node>


    <node name="intersection" pkg="four_way_stop" type="four_way_stop_node" output="screen">
        
    </node>

    <node name="rqt_gui" pkg="rqt_gui" type="rqt_gui"/>
</launch>
